tests/introspection/ApiCtorInitTest.cs

```diff
@@ -68,6 +68,13 @@ protected virtual bool Skip (Type type)
 				return true;
 			case "NEPacketTunnelProvider":
 				return true;
+			// On iOS 14 (beta 4) we get: [NISimulator] To simulate Nearby Interaction distance and direction, launch two or more simulators and
+			// move the simulator windows around the screen.
```

Comment:
This sounds like a great reason to request a bigger screen :smile:

Reply:
I got you covered https://www.samsung.com/us/computing/monitors/gaming/49--odyssey-g9-gaming-monitor-lc49g95tssnxza/ and if that is not enough you can get 2 of these https://www.staples.com/LG-86BH5C-B-86-LED-Display/product_IM12U5216

Reply:
👀 Looking for bigger apartment listings to accommodate an 86" monitor....